src/video/cocoa/SDL_cocoaopengl.m

```diff
@@ -43,6 +43,18 @@
 #pragma clang diagnostic ignored "-Wdeprecated-declarations"
 #endif
 
+static SDL_bool SDL_opengl_sync_dispatch = SDL_FALSE;
+
+static void SDLCALL
+SDL_OpenGLSyncDispatchChanged(void *userdata, const char *name, const char *oldValue, const char *hint)
+{
+    if (hint && *hint == '1') {
```

Comment:
Use `SDL_GetStringBoolean()` here

Reply:
Ah right, was looking at this [use case](https://github.com/libsdl-org/SDL/blob/8f5932dc2e3e5c4fadfd1d484b8fc5cd4e12639e/src/joystick/SDL_joystick.c#L216-L224) and forgot to double-check the one you mentioned.

Reply:
Ah right, was looking at this [use case](https://github.com/libsdl-org/SDL/blob/8f5932dc2e3e5c4fadfd1d484b8fc5cd4e12639e/src/joystick/SDL_joystick.c#L216-L224) and forgot to double-check the one you mentioned.

Reply:
Ah, thanks for catching that. Fixed!